src/webapp/cypress/integration/dashboard/org-manage.js

```diff
@@ -0,0 +1,40 @@
+describe(`Dashboard Org pages`, () => {
+
+    describe('an organi owner', () => {
```

Comment:
`organi` should be `org`

Reply:
Do we need a double nested describe?

Reply:
Technically no - I had written a series of tests that were "an organization owner should" then I had some tests as "a user should" then some tests as "an admin should" to prove the functionailty of the page against various auth roles. I thought it made it easier to see the breakdown if I could collapse all the org tests into a single line but maybe it's overkill - there only ended up being 2 tests in that section. I kind of like it- but if you want I can remove it and just start each it() with the same string instead

Reply:
That works well.